io/disk/port_bounce.py

```diff
@@ -191,23 +199,49 @@ def verify_port_enable(self, test_ports):
                 msg = "Port %s is failed to enable" % port
                 self.failure_list[port] = msg
 
-    def verify_port_toggle_host(self, status):
-        """
-        Verifies port enable/disable status change in host
-        side for corresponding fc adapter
-        """
-        for bus_id in self.pci_bus_addrs:
-            pci_class = pci.get_pci_class_name(bus_id)
-            intf = pci.get_interfaces_in_pci_address(bus_id, pci_class)[-1]
-            state = genio.read_file("/sys/class/fc_host/%s/port_state"
-                                    % intf).rstrip("\n")
-            self.log.info("Host bus: %s, state: %s", bus_id, state)
-            if state == status:
-                self.log.info("bus: %s, port status %s got reflected",
-                              bus_id, state)
-            else:
-                self.fail("port state not changed in host expected state: %s, \
-                          actual_state: %s", status, state)
+    def mpath_fail_check(self, test_ports):
+        '''
+        checking mpath disk status after disabling the switch port
+        '''
+        total_mpaths = 0
+        count = 0
+        wwids = multipath.get_multipath_wwids()
+        for wwid in wwids:
+            if wwid not in process.system_output('multipath -ll',
```

Comment:
@Naresh-ibm I think its better to run command outside of loop and just compare it in the loop.